lib/keycloak/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.7'
+
+services:
+  keycloak:
+    image: jboss/keycloak:9.0.2
+    container_name: keycloak
+    command: ["-b", "0.0.0.0", "-Dkeycloak.migration.strategy=IGNORE_EXISTING"]
+    ports:
+      - "${KC_LOCAL_PORT}:8081"
+    env_file:
+      - ./volumes/secrets.env
+    #networks:
+    #  - ${DOCKER_NET}
+    volumes:
+      - ./volumes/keycloak-db:/opt/jboss/keycloak/standalone/data
```

Comment:
please nest volumes in tmp/data, or create volumes via `docker volume create`. remember that local directories are not scalable so managed volumes/persistent volume claims offer a more flexible solution.